prql-compiler/tests/integration-rdbms/conf/my.cnf

```diff
@@ -1,2 +1,2 @@
 [mysqld]
-secure-file-priv= /tmp/chinook/
+secure-file-priv = ""
```

Comment:
Low priority question — but does this mean we can remove this file now?

Reply:
We need to set `secure-file-priv=""` either in the configuration file or in the command.
Perhaps it is more concise to write the command in the compose file, so I changed it in #2359.